src/flick/user/models.py

```diff
@@ -33,8 +34,11 @@ def num_notifs(self):
             Q(to_user=self) & ~(Q(notif_type="friend_request") & Q(friend_request_accepted=False))
         )
         unviewed_notifs = notifs.filter(Q(created_at__gt=self.notif_time_viewed)) if self.notif_time_viewed else notifs
-
-        return unviewed_notifs.count() + PrivateSuggestion.objects.filter(to_user=self.user).count()
+        suggests = PrivateSuggestion.objects.filter(to_user=self.user)
+        unviewed_suggests = (
+            suggests.filter(Q(created_at__gt=self.suggest_time_viewed)) if self.suggest_time_viewed else suggests
```

Comment:
is the `gt` stuff mean greater? lol interesting syntax

Reply:
YES!! HAHA